### DISCUSSION TITLE
JVM runtime metrics

### DISCUSSION THREAD

**Discussion by cxhansen** [open-telemetry/opentelemetry-java-instrumentation#2141] [Q&A]
Currently, there are very few [runtime metrics in the project](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/runtime-metrics/library/src/main/java/io/opentelemetry/instrumentation/runtimemetrics). Is that because the metrics API isn't frozen and so people are concerned that it's a moving target, or is just waiting for more contributions? Thanks!

**Comment by trask** [comment]
Hey @cxhansen, primarily just because focus has been on traces so far. We would definitely welcome metrics contributions!
